src/hb-aat-layout.cc

```diff
@@ -301,3 +309,29 @@ _hb_aat_language_get (hb_face_t *face,
 {
   return _get_ltag (face).get_language (i);
 }
+
+/**
+ * hb_aat_get_feature_settings:
+ * @face:            a font face.
+ * @identifier:      aat feature id you are querying.
```

Comment:
How would this API allow querying all features, instead of a specific known one / feature id?

Reply:
Revealing index of `feat` sound like exposing something client shouldn't be aware, I am not sure. Maybe a client that wants all the options can scan the table using different numbers (is that the usual case?). It is already sorted by feature type so that won't make much difference anyway.